test/apps/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/test)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+    $<TARGET_PROPERTY:teca_alg,INTERFACE_INCLUDE_DIRECTORIES>
+    $<TARGET_PROPERTY:teca_io,INTERFACE_INCLUDE_DIRECTORIES>
+    )
+
+teca_add_test(test_deeplabv3p_ar_detect_app
+    LIBS teca_io teca_alg
+    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/test_deeplabv3p_ar_detect_app.sh
+    ${CMAKE_BINARY_DIR}/${BIN_PREFIX}
+    --pytorch_deeplab_model ${TECA_DATA_ROOT}/cascade_deeplab_IVT.pt
+    --input_regex "${TECA_DATA_ROOT}/ARTMIP_MERRA_2D.*\.nc$"
+    --filename_time_template "ARTMIP_MERRA_2D_%Y%m%d_%H.nc"
+    --t_axis_variable ''
+    --t_units "days since 1979-01-01 00:00:00"
+    --output_file "${TECA_DATA_ROOT}/test_deeplabv3p_ar_detect_app_output.nc"
```

Comment:
Please have the app write intermediate results to the testing directory(cwd) rather than TECA_DATA_ROOT